src/node.cc

```diff
@@ -4502,6 +4529,8 @@ int Start(int argc, char** argv) {
   Init(&argc, const_cast<const char**>(argv), &exec_argc, &exec_argv);
 
 #if HAVE_OPENSSL
+  if (secure_getenv("NODE_EXTRA_CA_CERTS"))
```

Comment:
You can write this as `if (const char* var = secure_getenv("NODE_EXTRA_CA_CERTS"))` to avoid doing the lookup twice.